.github/ISSUE_TEMPLATE/BUG_REPORT.yml

```diff
@@ -0,0 +1,134 @@
+name: 🐞 Bug Report
+description: File a bug report if something is not working as expected.
+labels: []
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Before getting started, make sure you are in the right place by
+        ensuring that filing a bug report is the correct course of
+        action for your issue.
+
+        * For general technical questions, post a question on
+        [StackOverflow](http://stackoverflow.com/) with the `firebase` tag.
+
+        * For backend, console, and other non-SDK issues, reach out to
+        [Firebase Support](https://firebase.google.com/support/) for help.
+
+        If your issue does not fall under the above categories and instead
+        involves code in this repository not working as expected, please
+        complete and submit this bug report. Thank you!
+  - type: textarea
+    attributes:
+      label: Description
+      description: |
+        Describe the issue. Explain what you _expected_ to happen and what
+        _actually_ happened.
+    validations:
+      required: true
```

Comment:
One thing to note for reviewers, the [fields marked with the red asterisks are required to submit the form](https://github.com/ncooke3/firebase-ios-sdk/issues/new?assignees=&labels=&template=BUG_REPORT.yml). I put some thought into whether a field should be required or not but LMK what you think.